tests_integration/cli/config/editorconfig/.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[*]
```

Comment:
We should test a more specific glob as well, with different settings, like `[lib/**.js]` (copied from http://editorconfig.org/).